src/lexical-elements.rst

```diff
@@ -343,7 +343,7 @@ The following names are used when referring to :t:`[punctuator]s`:
      - Equals
```

Comment:
How are those operators selected, should we have all punctuation?
= assignment
or "//" comment?

Reply:
no I don't think we gain anything from adding this for all. The changes here were for completeness sake as the other operators of a similar group have the token names and semantic aliases